server/mailrepository/mailrepository-blob/src/main/scala/org/apache/james/mailrepository/blob/BlobMailRepository.scala

```diff
@@ -138,12 +135,12 @@ object BlobMailRepository {
   }
 }
 
-class BlobMailRepository(val blobStore: BlobStoreDAO,
+class BlobMailRepository @Inject() (val blobStore: BlobStoreDAO,
```

Comment:
Why add @Inject here ?

Reply:
Cause MemoryMailRepositoryStore would need it.